packages/api/src/providers/tenovi.ts

```diff
@@ -132,16 +137,17 @@ export class Tenovi extends Provider {
           connectedDevices,
           user: connectedUser.dataValues,
           level: "info",
+          cxName: xTenoviClientName,
         },
       });
       throw new Error(`Device ${deviceId} not found for this user.`);
```

Comment:
I know this was here before, but let's not make error messages dynamic (Sentry and grouping). Could move to `MetriportError("message...", undefined, { deviceId })`